Cargo.toml

```diff
@@ -61,7 +61,7 @@ default = ["grid"]
 browserslist = ["browserslist-rs"]
 cli = ["clap", "serde_json", "browserslist", "jemallocator"]
 grid = []
-serde = ["smallvec/serde", "cssparser/serde"]
+with-serde = ["serde", "smallvec/serde", "cssparser/serde"]
```

Comment:
This is a breaking change.

We could make it non-breaking by enabling the `with-serde` feature by default and having a `serde = []` feature, but that would enable `small-vec/serde` and `cssparser/serde`, which are not currently enabled by default.

It looks like `lightningcss` is on an alpha train. 
Is a small breaking change here acceptable?

Unless `serde` is commonly used for some reason, I think it makes sense to have it be disabled by default.

---

(Using the feature flag name "serde" as `serde = ["serde", "smallvec/serde", "cssparser/serde"]` wasn't possible since you can't define a feature flag with the same name as a dep.)

Reply:
I was able to rename it back to serde by doing this:

```
serde = ["dep:serde", "smallvec/serde", "cssparser/serde"]
```